### DISCUSSION TITLE
Considering remove git history in the generated project

### DISCUSSION THREAD

**Discussion by hoangnguyen92dn** [nimblehq/flutter_templates#166] [RFCs]
## Issue

After generating a new project from `flutter_templates`, it's annoying since the git history still remains in the generated project and the project is not clean and neat enough before ready to use for development. The generated project should be ready for development once the generating process is completed.

## Solution

Remove git folder to avoid showing git history on the generated project

## Who Benefits?

Developers

## What's Next?

- [ ] Address new issue to tackle this

**Comment by luongvo** [comment]
@hoangnguyen92dn We're planning to migrate the existing Python script to Mason. We will re-check this issue when working on that EPIC 👌

**Comment by luongvo** [comment]
@hoangnguyen92dn With the new project-generating script, we will `export the generated project into a new project folder` as below. That folder will be clean without git setup from the main Flutter templates. Please verify this update then we can close this RFC.

https://github.com/nimblehq/flutter-templates/tree/develop/sample

<img width="373" alt="image" src="https://github.com/nimblehq/flutter-templates/assets/16315358/1af49567-792c-424c-bed0-865a50963a40">


Thank you 🙏

**Reply by hoangnguyen92dn** [reply]
@luongvo Looks good 👍

**Reply by luongvo** [reply]
@hoangnguyen92dn thank you. It's working properly now, no git history in the output folder 👌 ... closing
